mbf_abstract_nav/include/mbf_abstract_nav/controller_action.h

```diff
@@ -66,6 +66,14 @@ class ControllerAction :
 
   void run(GoalHandle &goal_handle, AbstractControllerExecution &execution);
 
+protected:
+  void fillExePathFeedback(
```

Comment:
Better call it `publishExePathFeedback()` and use the `goal_handle` as parameter.